index.html

```diff
@@ -2295,10 +2289,20 @@ <h3 id="applying">
                   <code>undefined</code>, move onto the next item if any are
                   left.
                   </li>
-                  <li>Otherwise, let <var>application</var> be an object with
-                  properties <code>platform</code>, <code>id</code>,
-                  <code>url</code> respectively set to <var>platform</var>,
-                  <var>id</var> and <var>url</var>.
+                  <li>Let <var>minVersion</var> be the result of running the
+                  <a>steps for processing the <code>min_version</code> member
+                  of an application</a> with <var>potential application</var>.
+                  </li>
+                  <li>Let <var>fingerprints</var> be the result of running the
+                  <a>steps for processing the <code>fingerprints</code> member
+                  of an application</a> with <var>potential application</var>.
+                  </li>
+                  <li>Let <var>application</var> be an object with properties
+                  <code>platform</code>, <code>id</code>, <code>url</code>,
+                  <code>min_version</code>, and <code>fingerprints</code>
```

Comment:
FYI I think the normal way of writing it is "x, y and z, respectively, set to..." (comma before and after respectively). But it doesn't matter.